### PR TITLE
logout up for review

### PR DESCRIPTION
**Pull request by ssoonmi**

### CODE REVIEW

File: client/src/pages/PersonalModalButton.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  render() {
+    return (
+      <>
+        <button className='personalInfo' onClick={this.handleClick}><svg width="1em" height="1em" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+          <path fillRule="evenodd" d="M3 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H3zm5-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6z" />
+        </svg></button>
+        {this.state.showPersonalModal ?
+          <PersonalModal /> :
+          null
+        }
```
Comment (ssoonmi) [inline client/src/pages/PersonalModalButton.js:30]: Very nice!!!

**Comment by ssoonmi** [comment]
- [x] Should be able to click outside of the dropdown menu and close it. Hint: Make an element that has an onClick that will close the modal dropdown menu. Style it to be as big as the whole page and render it behind the dropdown menu.

File: client/src/pages/Dashboard.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PersonalModalButton from './PersonalModalButton';
+import './dashboard.css';
+
+
+export default function Dashboard() {
```
Comment (ssoonmi) [inline client/src/pages/Dashboard.js:9]: Here, make sure to redirect the user if there is no user logged in

File: client/src/pages/PersonalModalButton.js (hunk 1 above)
Comment (shaieleigh) [reply client/src/pages/PersonalModalButton.js:30]: thanks

File: client/src/pages/Dashboard.js (hunk 2 above)
Comment (shaieleigh) [reply client/src/pages/Dashboard.js:9]: ok

**Pull request merged**